teamcity: failed tests on master: test/TestSystemZoneConfigs

The following tests appear to have failed:

[#312112](https://teamcity.cockroachdb.com/viewLog.html?buildId=312112):

```
--- FAIL: test/TestSystemZoneConfigs (45.860s)
soon.go:43: storage/client_replica_test.go:1570, condition failed to evaluate within 45s: got 43 replicas, want 44; details: map[12:3 13:3 4:3 3:3 6:3 10:3 1:4 7:3 8:3 11:3 5:3 9:3 14:3 2:3]
------- Stdout: -------
W170801 22:39:22.381238 63187 server/server.go:298  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170801 22:39:22.381501 63187 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170801 22:39:22.386162 63187 server/config.go:521  [n?] 1 storage engine initialized
I170801 22:39:22.386187 63187 server/config.go:523  [n?] RocksDB cache size: 512 MiB
I170801 22:39:22.386197 63187 server/config.go:523  [n?] store 0: in-memory, size 100 MiB
I170801 22:39:22.389696 63187 server/node.go:335  [n?] **** cluster 9b8fee7b-75ad-4153-8499-ef71dbb0a46c has been created
I170801 22:39:22.389739 63187 server/server.go:780  [n?] **** add additional nodes by specifying --join=127.0.0.1:57613
I170801 22:39:22.390421 63187 storage/store.go:1271  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170801 22:39:22.390459 63187 server/node.go:421  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:0 WritesPerSecond:0}
I170801 22:39:22.390490 63187 server/node.go:319  [n1] node ID 1 initialized
I170801 22:39:22.390539 63187 gossip/gossip.go:295  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:57613" > attrs:<> locality:<> 
I170801 22:39:22.390684 63187 storage/stores.go:293  [n1] read 0 node addresses from persistent storage
I170801 22:39:22.390763 63187 server/node.go:556  [n1] connecting to gossip network to verify cluster ID...
I170801 22:39:22.390777 63187 server/node.go:581  [n1] node connected via gossip and verified as part of cluster "9b8fee7b-75ad-4153-8499-ef71dbb0a46c"
I170801 22:39:22.390794 63187 server/node.go:362  [n1] node=1: started with [=] engine(s) and attributes []
I170801 22:39:22.390852 63187 sql/executor.go:416  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:57613}
I170801 22:39:22.393809 62878 storage/replica_command.go:2667  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I170801 22:39:22.394245 63187 server/server.go:859  [n1] starting https server at 127.0.0.1:55341
I170801 22:39:22.394265 63187 server/server.go:860  [n1] starting grpc/postgres server at 127.0.0.1:57613
I170801 22:39:22.394273 63187 server/server.go:861  [n1] advertising CockroachDB node at 127.0.0.1:57613
W170801 22:39:22.394291 63187 sql/jobs/registry.go:148  [n1] unable to get node liveness: node not in the liveness table
W170801 22:39:22.394297 63187 sql/jobs/registry.go:273  [n1] canceling all jobs due to liveness failure
I170801 22:39:22.410613 63187 sql/event_log.go:99  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN "uniqueID" SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170801 22:39:22.418147 62878 storage/replica_command.go:2667  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/tsd [r3]
I170801 22:39:22.424400 62878 storage/replica_command.go:2667  [split,n1,s1,r3/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r4]
I170801 22:39:22.425322 63187 sql/lease.go:357  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-08-01 22:39:22.424438414 +0000 UTC
I170801 22:39:22.432130 62878 storage/replica_command.go:2667  [split,n1,s1,r4/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r5]
I170801 22:39:22.438327 62878 storage/replica_command.go:2667  [split,n1,s1,r5/1:/{Table/System���-Max}] initiating a split of this range at key /Table/11 [r6]
I170801 22:39:22.444055 62878 storage/replica_command.go:2667  [split,n1,s1,r6/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r7]
I170801 22:39:22.449209 62878 storage/replica_command.go:2667  [split,n1,s1,r7/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r8]
I170801 22:39:22.458290 63187 server/server.go:999  [n1] done ensuring all necessary migrations have run
I170801 22:39:22.458324 63187 server/server.go:1001  [n1] serving sql connections
I170801 22:39:22.459322 62878 storage/replica_command.go:2667  [split,n1,s1,r8/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r9]
I170801 22:39:22.461938 63629 sql/event_log.go:99  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:57613} Attrs: Locality:} ClusterID:9b8fee7b-75ad-4153-8499-ef71dbb0a46c StartedAt:1501627162390780880 LastUp:1501627162390780880}
I170801 22:39:22.469556 62878 storage/replica_command.go:2667  [split,n1,s1,r9/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r10]
I170801 22:39:22.479602 62878 storage/replica_command.go:2667  [split,n1,s1,r10/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r11]
I170801 22:39:22.487911 62878 storage/replica_command.go:2667  [split,n1,s1,r11/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r12]
I170801 22:39:22.492792 62878 storage/replica_command.go:2667  [split,n1,s1,r12/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r13]
I170801 22:39:22.497897 62878 storage/replica_command.go:2667  [split,n1,s1,r13/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r14]
W170801 22:39:22.512667 63187 server/server.go:298  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170801 22:39:22.512852 63187 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
W170801 22:39:22.513392 63187 gossip/gossip.go:1204  [n?] no incoming or outgoing connections
I170801 22:39:22.515509 63187 server/config.go:521  [n?] 1 storage engine initialized
I170801 22:39:22.515530 63187 server/config.go:523  [n?] RocksDB cache size: 512 MiB
I170801 22:39:22.515541 63187 server/config.go:523  [n?] store 0: in-memory, size 100 MiB
I170801 22:39:22.515583 63187 server/server.go:782  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I170801 22:39:22.524329 63954 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:57613
I170801 22:39:22.524557 63913 gossip/server.go:232  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:42478}
I170801 22:39:22.524795 63187 server/node.go:408  [n?] store [n0,s0] not bootstrapped
I170801 22:39:22.524814 63187 storage/stores.go:293  [n?] read 0 node addresses from persistent storage
I170801 22:39:22.524850 63187 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170801 22:39:22.524862 63187 server/node.go:556  [n?] connecting to gossip network to verify cluster ID...
I170801 22:39:22.524873 63187 server/node.go:581  [n?] node connected via gossip and verified as part of cluster "9b8fee7b-75ad-4153-8499-ef71dbb0a46c"
I170801 22:39:22.526179 63187 kv/dist_sender.go:362  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170801 22:39:22.527268 63187 server/node.go:312  [n?] new node allocated ID 2
I170801 22:39:22.527312 63187 gossip/gossip.go:295  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:42478" > attrs:<> locality:<> 
I170801 22:39:22.527408 63187 server/node.go:362  [n2] node=2: started with [=] engine(s) and attributes []
I170801 22:39:22.527497 63187 sql/executor.go:416  [n2] creating distSQLPlanner with address {tcp 127.0.0.1:42478}
I170801 22:39:22.528891 63638 storage/stores.go:312  [n1] wrote 1 node addresses to persistent storage
I170801 22:39:22.533310 63187 server/server.go:859  [n2] starting https server at 127.0.0.1:42412
I170801 22:39:22.533369 63187 server/server.go:860  [n2] starting grpc/postgres server at 127.0.0.1:42478
I170801 22:39:22.533376 63187 server/server.go:861  [n2] advertising CockroachDB node at 127.0.0.1:42478
W170801 22:39:22.533385 63187 sql/jobs/registry.go:148  [n2] unable to get node liveness: node not in the liveness table
W170801 22:39:22.533390 63187 sql/jobs/registry.go:273  [n2] canceling all jobs due to liveness failure
I170801 22:39:22.533618 64005 server/node.go:537  [n2] bootstrapped store [n2,s2]
I170801 22:39:22.534171 63187 server/server.go:999  [n2] done ensuring all necessary migrations have run
I170801 22:39:22.534190 63187 server/server.go:1001  [n2] serving sql connections
I170801 22:39:22.541250 63961 sql/event_log.go:99  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:42478} Attrs: Locality:} ClusterID:9b8fee7b-75ad-4153-8499-ef71dbb0a46c StartedAt:1501627162527382590 LastUp:1501627162527382590}
W170801 22:39:22.541949 63187 server/server.go:298  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170801 22:39:22.542938 63187 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
W170801 22:39:22.545232 63187 gossip/gossip.go:1204  [n?] no incoming or outgoing connections
I170801 22:39:22.547665 63187 server/config.go:521  [n?] 1 storage engine initialized
I170801 22:39:22.547699 63187 server/config.go:523  [n?] RocksDB cache size: 512 MiB
I170801 22:39:22.547708 63187 server/config.go:523  [n?] store 0: in-memory, size 100 MiB
I170801 22:39:22.547776 63187 server/server.go:782  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I170801 22:39:22.554315 64150 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:57613
I170801 22:39:22.554650 63873 gossip/server.go:232  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:58372}
I170801 22:39:22.555103 63187 server/node.go:408  [n?] store [n0,s0] not bootstrapped
I170801 22:39:22.555128 63187 storage/stores.go:293  [n?] read 0 node addresses from persistent storage
I170801 22:39:22.555157 63187 storage/stores.go:312  [n?] wrote 2 node addresses to persistent storage
I170801 22:39:22.555170 63187 server/node.go:556  [n?] connecting to gossip network to verify cluster ID...
I170801 22:39:22.555188 63187 server/node.go:581  [n?] node connected via gossip and verified as part of cluster "9b8fee7b-75ad-4153-8499-ef71dbb0a46c"
I170801 22:39:22.556481 63187 kv/dist_sender.go:362  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170801 22:39:22.557480 63187 server/node.go:312  [n?] new node allocated ID 3
I170801 22:39:22.557536 63187 gossip/gossip.go:295  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:58372" > attrs:<> locality:<> 
I170801 22:39:22.557601 63187 server/node.go:362  [n3] node=3: started with [=] engine(s) and attributes []
I170801 22:39:22.557647 63187 sql/executor.go:416  [n3] creating distSQLPlanner with address {tcp 127.0.0.1:58372}
I170801 22:39:22.558067 64132 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170801 22:39:22.558365 63647 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170801 22:39:22.558600 64229 server/node.go:537  [n3] bootstrapped store [n3,s3]
I170801 22:39:22.560216 63187 server/server.go:859  [n3] starting https server at 127.0.0.1:43894
I170801 22:39:22.560239 63187 server/server.go:860  [n3] starting grpc/postgres server at 127.0.0.1:58372
I170801 22:39:22.560251 63187 server/server.go:861  [n3] advertising CockroachDB node at 127.0.0.1:58372
W170801 22:39:22.560266 63187 sql/jobs/registry.go:148  [n3] unable to get node liveness: node not in the liveness table
W170801 22:39:22.560276 63187 sql/jobs/registry.go:273  [n3] canceling all jobs due to liveness failure
I170801 22:39:22.561081 63187 server/server.go:999  [n3] done ensuring all necessary migrations have run
I170801 22:39:22.561103 63187 server/server.go:1001  [n3] serving sql connections
W170801 22:39:22.568916 63187 server/server.go:298  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
W170801 22:39:22.569105 63187 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
W170801 22:39:22.569503 63187 gossip/gossip.go:1204  [n?] no incoming or outgoing connections
I170801 22:39:22.569839 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r11/1:/Table/1{6-7}] generated preemptive snapshot 00485209 at index 16
I170801 22:39:22.572551 63187 server/config.go:521  [n?] 1 storage engine initialized
I170801 22:39:22.572588 63187 server/config.go:523  [n?] RocksDB cache size: 512 MiB
I170801 22:39:22.572612 63187 server/config.go:523  [n?] store 0: in-memory, size 100 MiB
I170801 22:39:22.572646 63187 server/server.go:782  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I170801 22:39:22.574341 63994 sql/event_log.go:99  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:58372} Attrs: Locality:} ClusterID:9b8fee7b-75ad-4153-8499-ef71dbb0a46c StartedAt:1501627162557589672 LastUp:1501627162557589672}
I170801 22:39:22.579953 64137 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:57613
I170801 22:39:22.580216 64435 gossip/server.go:232  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:46420}
I170801 22:39:22.580277 63503 storage/store.go:3486  [replicate,n1,s1,r11/1:/Table/1{6-7}] streamed snapshot to (n2,s2):?: kv pairs: 9, log entries: 6, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.580654 63187 server/node.go:408  [n?] store [n0,s0] not bootstrapped
I170801 22:39:22.580670 63187 storage/stores.go:293  [n?] read 0 node addresses from persistent storage
I170801 22:39:22.580707 63187 storage/stores.go:312  [n?] wrote 3 node addresses to persistent storage
I170801 22:39:22.580712 63187 server/node.go:556  [n?] connecting to gossip network to verify cluster ID...
I170801 22:39:22.580723 63187 server/node.go:581  [n?] node connected via gossip and verified as part of cluster "9b8fee7b-75ad-4153-8499-ef71dbb0a46c"
I170801 22:39:22.580740 64381 storage/replica_raftstorage.go:718  [n2,s2,r11/?:{-}] applying preemptive snapshot at index 16 (id=00485209, encoded size=4049, 1 rocksdb batches, 6 log entries)
I170801 22:39:22.581224 64381 storage/replica_raftstorage.go:726  [n2,s2,r11/?:/Table/1{6-7}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.581544 63503 storage/replica_command.go:3600  [replicate,n1,s1,r11/1:/Table/1{6-7}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r11:/Table/1{6-7} [(n1,s1):1, next=2]
I170801 22:39:22.581952 63187 kv/dist_sender.go:362  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170801 22:39:22.582933 63187 server/node.go:312  [n?] new node allocated ID 4
I170801 22:39:22.583046 63187 gossip/gossip.go:295  [n4] NodeDescriptor set to node_id:4 address:<network_field:"tcp" address_field:"127.0.0.1:46420" > attrs:<> locality:<> 
I170801 22:39:22.583095 63187 server/node.go:362  [n4] node=4: started with [=] engine(s) and attributes []
I170801 22:39:22.583142 63187 sql/executor.go:416  [n4] creating distSQLPlanner with address {tcp 127.0.0.1:46420}
I170801 22:39:22.584527 64471 storage/stores.go:312  [n1] wrote 3 node addresses to persistent storage
I170801 22:39:22.584656 64485 storage/replica.go:2937  [n1,s1,r11/1:/Table/1{6-7}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170801 22:39:22.584818 64145 storage/stores.go:312  [n3] wrote 3 node addresses to persistent storage
I170801 22:39:22.584844 64472 storage/stores.go:312  [n2] wrote 3 node addresses to persistent storage
I170801 22:39:22.585341 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r8/1:/Table/1{3-4}] generated preemptive snapshot 2b8e5d8e at index 34
I170801 22:39:22.586043 64429 server/node.go:537  [n4] bootstrapped store [n4,s4]
I170801 22:39:22.586617 63187 server/server.go:859  [n4] starting https server at 127.0.0.1:43789
I170801 22:39:22.586634 63187 server/server.go:860  [n4] starting grpc/postgres server at 127.0.0.1:46420
I170801 22:39:22.586658 63187 server/server.go:861  [n4] advertising CockroachDB node at 127.0.0.1:46420
W170801 22:39:22.586666 63187 sql/jobs/registry.go:148  [n4] unable to get node liveness: node not in the liveness table
W170801 22:39:22.586671 63187 sql/jobs/registry.go:273  [n4] canceling all jobs due to liveness failure
I170801 22:39:22.587254 64458 storage/raft_transport.go:451  [n2] raft transport stream to node 1 established
I170801 22:39:22.588266 63187 server/server.go:999  [n4] done ensuring all necessary migrations have run
I170801 22:39:22.588292 63187 server/server.go:1001  [n4] serving sql connections
W170801 22:39:22.598467 63187 server/server.go:298  [n?] all stores are configured as in-memory stores, so not setting up a temporary store. Queries with working set larger than memory will fail
I170801 22:39:22.598643 63503 storage/store.go:3486  [replicate,n1,s1,r8/1:/Table/1{3-4}] streamed snapshot to (n3,s3):?: kv pairs: 97, log entries: 24, rate-limit: 8.0 MiB/sec, 1ms
W170801 22:39:22.598660 63187 server/status/runtime.go:109  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170801 22:39:22.598813 64677 storage/replica_raftstorage.go:718  [n3,s3,r8/?:{-}] applying preemptive snapshot at index 34 (id=2b8e5d8e, encoded size=34676, 1 rocksdb batches, 24 log entries)
W170801 22:39:22.599048 63187 gossip/gossip.go:1204  [n?] no incoming or outgoing connections
I170801 22:39:22.599720 64677 storage/replica_raftstorage.go:726  [n3,s3,r8/?:/Table/1{3-4}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.600293 64645 sql/event_log.go:99  [n4] Event: "node_join", target: 4, info: {Descriptor:{NodeID:4 Address:{NetworkField:tcp AddressField:127.0.0.1:46420} Attrs: Locality:} ClusterID:9b8fee7b-75ad-4153-8499-ef71dbb0a46c StartedAt:1501627162583083919 LastUp:1501627162583083919}
I170801 22:39:22.600320 63503 storage/replica_command.go:3600  [replicate,n1,s1,r8/1:/Table/1{3-4}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r8:/Table/1{3-4} [(n1,s1):1, next=2]
I170801 22:39:22.600957 63187 server/config.go:521  [n?] 1 storage engine initialized
I170801 22:39:22.601050 63187 server/config.go:523  [n?] RocksDB cache size: 512 MiB
I170801 22:39:22.601059 63187 server/config.go:523  [n?] store 0: in-memory, size 100 MiB
I170801 22:39:22.601091 63187 server/server.go:782  [n?] no stores bootstrapped and --join flag specified, awaiting init command.
I170801 22:39:22.604109 64686 storage/replica.go:2937  [n1,s1,r8/1:/Table/1{3-4}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170801 22:39:22.604732 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r12/1:/Table/1{7-8}] generated preemptive snapshot 990e033e at index 16
I170801 22:39:22.605407 63503 storage/store.go:3486  [replicate,n1,s1,r12/1:/Table/1{7-8}] streamed snapshot to (n2,s2):?: kv pairs: 9, log entries: 6, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.605640 64754 storage/replica_raftstorage.go:718  [n2,s2,r12/?:{-}] applying preemptive snapshot at index 16 (id=990e033e, encoded size=4045, 1 rocksdb batches, 6 log entries)
I170801 22:39:22.606089 64754 storage/replica_raftstorage.go:726  [n2,s2,r12/?:/Table/1{7-8}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.606145 64657 storage/raft_transport.go:451  [n3] raft transport stream to node 1 established
I170801 22:39:22.607234 63503 storage/replica_command.go:3600  [replicate,n1,s1,r12/1:/Table/1{7-8}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r12:/Table/1{7-8} [(n1,s1):1, next=2]
I170801 22:39:22.609294 64708 gossip/client.go:129  [n?] started gossip client to 127.0.0.1:57613
I170801 22:39:22.609571 64694 gossip/server.go:232  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:50803}
I170801 22:39:22.610203 63187 server/node.go:408  [n?] store [n0,s0] not bootstrapped
I170801 22:39:22.610225 63187 storage/stores.go:293  [n?] read 0 node addresses from persistent storage
I170801 22:39:22.610259 63187 storage/stores.go:312  [n?] wrote 4 node addresses to persistent storage
I170801 22:39:22.610272 63187 server/node.go:556  [n?] connecting to gossip network to verify cluster ID...
I170801 22:39:22.610293 63187 server/node.go:581  [n?] node connected via gossip and verified as part of cluster "9b8fee7b-75ad-4153-8499-ef71dbb0a46c"
I170801 22:39:22.611096 64781 storage/replica.go:2937  [n1,s1,r12/1:/Table/1{7-8}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170801 22:39:22.611522 63187 kv/dist_sender.go:362  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170801 22:39:22.611794 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r13/1:/Table/1{8-9}] generated preemptive snapshot d3f31ad9 at index 15
I170801 22:39:22.612371 63503 storage/store.go:3486  [replicate,n1,s1,r13/1:/Table/1{8-9}] streamed snapshot to (n3,s3):?: kv pairs: 9, log entries: 5, rate-limit: 8.0 MiB/sec, 0ms
I170801 22:39:22.612398 63187 server/node.go:312  [n?] new node allocated ID 5
I170801 22:39:22.612433 63187 gossip/gossip.go:295  [n5] NodeDescriptor set to node_id:5 address:<network_field:"tcp" address_field:"127.0.0.1:50803" > attrs:<> locality:<> 
I170801 22:39:22.612489 63187 server/node.go:362  [n5] node=5: started with [=] engine(s) and attributes []
I170801 22:39:22.612527 63187 sql/executor.go:416  [n5] creating distSQLPlanner with address {tcp 127.0.0.1:50803}
I170801 22:39:22.612617 64785 storage/replica_raftstorage.go:718  [n3,s3,r13/?:{-}] applying preemptive snapshot at index 15 (id=d3f31ad9, encoded size=3654, 1 rocksdb batches, 5 log entries)
I170801 22:39:22.612775 64694 gossip/server.go:283  [n1] refusing gossip from node 5 (max 3 conns); forwarding to 3 ({tcp 127.0.0.1:58372})
I170801 22:39:22.613047 64745 storage/stores.go:312  [n1] wrote 4 node addresses to persistent storage
I170801 22:39:22.613198 64708 gossip/client.go:134  [n5] closing client to node 1 (127.0.0.1:57613): received forward from node 1 to 3 (127.0.0.1:58372)
I170801 22:39:22.613252 64732 gossip/gossip.go:1218  [n5] node has connected to cluster via gossip
I170801 22:39:22.613320 64746 storage/stores.go:312  [n2] wrote 4 node addresses to persistent storage
I170801 22:39:22.613354 64732 storage/stores.go:312  [n5] wrote 4 node addresses to persistent storage
I170801 22:39:22.613426 64669 storage/stores.go:312  [n4] wrote 4 node addresses to persistent storage
I170801 22:39:22.613478 64718 storage/stores.go:312  [n3] wrote 4 node addresses to persistent storage
I170801 22:39:22.613936 64785 storage/replica_raftstorage.go:726  [n3,s3,r13/?:/Table/1{8-9}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170801 22:39:22.614315 63503 storage/replica_command.go:3600  [replicate,n1,s1,r13/1:/Table/1{8-9}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r13:/Table/1{8-9} [(n1,s1):1, next=2]
I170801 22:39:22.615216 64817 server/node.go:537  [n5] bootstrapped store [n5,s5]
I170801 22:39:22.618420 63187 server/server.go:859  [n5] starting https server at 127.0.0.1:48324
I170801 22:39:22.618459 63187 server/server.go:860  [n5] starting grpc/postgres server at 127.0.0.1:50803
I170801 22:39:22.618468 63187 server/server.go:861  [n5] advertising CockroachDB node at 127.0.0.1:50803
W170801 22:39:22.618483 63187 sql/jobs/registry.go:148  [n5] unable to get node liveness: node not in the liveness table
W170801 22:39:22.618490 63187 sql/jobs/registry.go:273  [n5] canceling all jobs due to liveness failure
I170801 22:39:22.621438 63187 server/server.go:999  [n5] done ensuring all necessary migrations have run
I170801 22:39:22.621492 63187 server/server.go:1001  [n5] serving sql connections
I170801 22:39:22.623506 64999 storage/replica.go:2937  [n1,s1,r13/1:/Table/1{8-9}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170801 22:39:22.624519 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r10/1:/Table/1{5-6}] generated preemptive snapshot dc8988b0 at index 19
I170801 22:39:22.627006 64885 sql/event_log.go:99  [n5] Event: "node_join", target: 5, info: {Descriptor:{NodeID:5 Address:{NetworkField:tcp AddressField:127.0.0.1:50803} Attrs: Locality:} ClusterID:9b8fee7b-75ad-4153-8499-ef71dbb0a46c StartedAt:1501627162612479212 LastUp:1501627162612479212}
I170801 22:39:22.627472 64695 gossip/client.go:129  [n5] started gossip client to 127.0.0.1:58372
I170801 22:39:22.634984 63503 storage/store.go:3486  [replicate,n1,s1,r10/1:/Table/1{5-6}] streamed snapshot to (n4,s4):?: kv pairs: 9, log entries: 9, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.635173 65078 storage/replica_raftstorage.go:718  [n4,s4,r10/?:{-}] applying preemptive snapshot at index 19 (id=dc8988b0, encoded size=5726, 1 rocksdb batches, 9 log entries)
I170801 22:39:22.635723 65078 storage/replica_raftstorage.go:726  [n4,s4,r10/?:/Table/1{5-6}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.636127 63503 storage/replica_command.go:3600  [replicate,n1,s1,r10/1:/Table/1{5-6}] change replicas (ADD_REPLICA (n4,s4):2): read existing descriptor r10:/Table/1{5-6} [(n1,s1):1, next=2]
I170801 22:39:22.639892 65122 storage/replica.go:2937  [n1,s1,r10/1:/Table/1{5-6}] proposing ADD_REPLICA((n4,s4):2): updated=[(n1,s1):1 (n4,s4):2] next=3
I170801 22:39:22.640750 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] generated preemptive snapshot 882f78b7 at index 26
I170801 22:39:22.642676 65083 storage/raft_transport.go:451  [n4] raft transport stream to node 1 established
I170801 22:39:22.651568 63503 storage/store.go:3486  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] streamed snapshot to (n5,s5):?: kv pairs: 50, log entries: 16, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.651794 65138 storage/replica_raftstorage.go:718  [n5,s5,r5/?:{-}] applying preemptive snapshot at index 26 (id=882f78b7, encoded size=26374, 1 rocksdb batches, 16 log entries)
I170801 22:39:22.652510 65138 storage/replica_raftstorage.go:726  [n5,s5,r5/?:/Table/{SystemCon���-11}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.652908 63503 storage/replica_command.go:3600  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] change replicas (ADD_REPLICA (n5,s5):2): read existing descriptor r5:/Table/{SystemConfigSpan/Start-11} [(n1,s1):1, next=2]
I170801 22:39:22.656658 65163 storage/replica.go:2937  [n1,s1,r5/1:/Table/{SystemCon���-11}] proposing ADD_REPLICA((n5,s5):2): updated=[(n1,s1):1 (n5,s5):2] next=3
I170801 22:39:22.657398 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r2/1:/System/{-tsd}] generated preemptive snapshot 9e055f14 at index 51
I170801 22:39:22.658543 63503 storage/store.go:3486  [replicate,n1,s1,r2/1:/System/{-tsd}] streamed snapshot to (n4,s4):?: kv pairs: 43, log entries: 13, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.658948 65188 storage/replica_raftstorage.go:718  [n4,s4,r2/?:{-}] applying preemptive snapshot at index 51 (id=9e055f14, encoded size=151795, 1 rocksdb batches, 13 log entries)
I170801 22:39:22.659750 65176 storage/raft_transport.go:451  [n5] raft transport stream to node 1 established
I170801 22:39:22.660268 65188 storage/replica_raftstorage.go:726  [n4,s4,r2/?:/System/{-tsd}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170801 22:39:22.660874 63503 storage/replica_command.go:3600  [replicate,n1,s1,r2/1:/System/{-tsd}] change replicas (ADD_REPLICA (n4,s4):2): read existing descriptor r2:/System/{-tsd} [(n1,s1):1, next=2]
I170801 22:39:22.664983 65127 storage/replica.go:2937  [n1,s1,r2/1:/System/{-tsd}] proposing ADD_REPLICA((n4,s4):2): updated=[(n1,s1):1 (n4,s4):2] next=3
I170801 22:39:22.665708 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r14/1:/{Table/19-Max}] generated preemptive snapshot fcb0fcce at index 11
I170801 22:39:22.666484 63503 storage/store.go:3486  [replicate,n1,s1,r14/1:/{Table/19-Max}] streamed snapshot to (n3,s3):?: kv pairs: 8, log entries: 1, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.666727 65201 storage/replica_raftstorage.go:718  [n3,s3,r14/?:{-}] applying preemptive snapshot at index 11 (id=fcb0fcce, encoded size=472, 1 rocksdb batches, 1 log entries)
I170801 22:39:22.667100 65201 storage/replica_raftstorage.go:726  [n3,s3,r14/?:/{Table/19-Max}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.667588 63503 storage/replica_command.go:3600  [replicate,n1,s1,r14/1:/{Table/19-Max}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r14:/{Table/19-Max} [(n1,s1):1, next=2]
I170801 22:39:22.671372 65250 storage/replica.go:2937  [n1,s1,r14/1:/{Table/19-Max}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170801 22:39:22.672090 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r9/1:/Table/1{4-5}] generated preemptive snapshot f2bf7cd7 at index 19
I170801 22:39:22.672686 63503 storage/store.go:3486  [replicate,n1,s1,r9/1:/Table/1{4-5}] streamed snapshot to (n4,s4):?: kv pairs: 9, log entries: 9, rate-limit: 8.0 MiB/sec, 0ms
I170801 22:39:22.672947 65133 storage/replica_raftstorage.go:718  [n4,s4,r9/?:{-}] applying preemptive snapshot at index 19 (id=f2bf7cd7, encoded size=5723, 1 rocksdb batches, 9 log entries)
I170801 22:39:22.673471 65133 storage/replica_raftstorage.go:726  [n4,s4,r9/?:/Table/1{4-5}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.673783 63503 storage/replica_command.go:3600  [replicate,n1,s1,r9/1:/Table/1{4-5}] change replicas (ADD_REPLICA (n4,s4):2): read existing descriptor r9:/Table/1{4-5} [(n1,s1):1, next=2]
I170801 22:39:22.677277 65254 storage/replica.go:2937  [n1,s1,r9/1:/Table/1{4-5}] proposing ADD_REPLICA((n4,s4):2): updated=[(n1,s1):1 (n4,s4):2] next=3
I170801 22:39:22.678053 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] generated preemptive snapshot c979298d at index 23
I170801 22:39:22.678746 63503 storage/store.go:3486  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] streamed snapshot to (n5,s5):?: kv pairs: 11, log entries: 13, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.679023 65147 storage/replica_raftstorage.go:718  [n5,s5,r4/?:{-}] applying preemptive snapshot at index 23 (id=c979298d, encoded size=10091, 1 rocksdb batches, 13 log entries)
I170801 22:39:22.679578 65147 storage/replica_raftstorage.go:726  [n5,s5,r4/?:/{System/tse-Table/System���}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.679908 63503 storage/replica_command.go:3600  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] change replicas (ADD_REPLICA (n5,s5):2): read existing descriptor r4:/{System/tse-Table/SystemConfigSpan/Start} [(n1,s1):1, next=2]
I170801 22:39:22.684222 65270 storage/replica.go:2937  [n1,s1,r4/1:/{System/tse-Table/System���}] proposing ADD_REPLICA((n5,s5):2): updated=[(n1,s1):1 (n5,s5):2] next=3
I170801 22:39:22.684886 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r3/1:/System/ts{d-e}] generated preemptive snapshot f1134398 at index 23
I170801 22:39:22.693148 63503 storage/store.go:3486  [replicate,n1,s1,r3/1:/System/ts{d-e}] streamed snapshot to (n2,s2):?: kv pairs: 1421, log entries: 3, rate-limit: 8.0 MiB/sec, 8ms
I170801 22:39:22.693492 65284 storage/replica_raftstorage.go:718  [n2,s2,r3/?:{-}] applying preemptive snapshot at index 23 (id=f1134398, encoded size=202893, 1 rocksdb batches, 3 log entries)
I170801 22:39:22.694879 65284 storage/replica_raftstorage.go:726  [n2,s2,r3/?:/System/ts{d-e}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170801 22:39:22.695440 63503 storage/replica_command.go:3600  [replicate,n1,s1,r3/1:/System/ts{d-e}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r3:/System/ts{d-e} [(n1,s1):1, next=2]
I170801 22:39:22.699022 65304 storage/replica.go:2937  [n1,s1,r3/1:/System/ts{d-e}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170801 22:39:22.699723 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r11/1:/Table/1{6-7}] generated preemptive snapshot 0ebeb4be at index 19
I170801 22:39:22.700474 62879 storage/store.go:3486  [replicate,n1,s1,r11/1:/Table/1{6-7}] streamed snapshot to (n5,s5):?: kv pairs: 10, log entries: 9, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.700685 65262 storage/replica_raftstorage.go:718  [n5,s5,r11/?:{-}] applying preemptive snapshot at index 19 (id=0ebeb4be, encoded size=6080, 1 rocksdb batches, 9 log entries)
I170801 22:39:22.701090 65262 storage/replica_raftstorage.go:726  [n5,s5,r11/?:/Table/1{6-7}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.701442 62879 storage/replica_command.go:3600  [replicate,n1,s1,r11/1:/Table/1{6-7}] change replicas (ADD_REPLICA (n5,s5):3): read existing descriptor r11:/Table/1{6-7} [(n1,s1):1, (n2,s2):2, next=3]
I170801 22:39:22.705571 65352 storage/replica.go:2937  [n1,s1,r11/1:/Table/1{6-7}] proposing ADD_REPLICA((n5,s5):3): updated=[(n1,s1):1 (n2,s2):2 (n5,s5):3] next=4
I170801 22:39:22.706821 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r6/1:/Table/1{1-2}] generated preemptive snapshot 6c73778b at index 16
I170801 22:39:22.707779 63503 storage/store.go:3486  [replicate,n1,s1,r6/1:/Table/1{1-2}] streamed snapshot to (n2,s2):?: kv pairs: 9, log entries: 6, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.708093 65286 storage/replica_raftstorage.go:718  [n2,s2,r6/?:{-}] applying preemptive snapshot at index 16 (id=6c73778b, encoded size=5299, 1 rocksdb batches, 6 log entries)
I170801 22:39:22.708542 65286 storage/replica_raftstorage.go:726  [n2,s2,r6/?:/Table/1{1-2}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.708962 63503 storage/replica_command.go:3600  [replicate,n1,s1,r6/1:/Table/1{1-2}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r6:/Table/1{1-2} [(n1,s1):1, next=2]
I170801 22:39:22.712701 65246 storage/replica.go:2937  [n1,s1,r6/1:/Table/1{1-2}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170801 22:39:22.713381 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot c073f023 at index 91
I170801 22:39:22.714138 63503 storage/store.go:3486  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n5,s5):?: kv pairs: 53, log entries: 28, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.714419 65357 storage/replica_raftstorage.go:718  [n5,s5,r1/?:{-}] applying preemptive snapshot at index 91 (id=c073f023, encoded size=16088, 1 rocksdb batches, 28 log entries)
I170801 22:39:22.715273 65357 storage/replica_raftstorage.go:726  [n5,s5,r1/?:/{Min-System/}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170801 22:39:22.715679 63503 storage/replica_command.go:3600  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n5,s5):2): read existing descriptor r1:/{Min-System/} [(n1,s1):1, next=2]
I170801 22:39:22.718675 65295 storage/replica.go:2937  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA((n5,s5):2): updated=[(n1,s1):1 (n5,s5):2] next=3
I170801 22:39:22.719422 63503 storage/replica_raftstorage.go:509  [replicate,n1,s1,r7/1:/Table/1{2-3}] generated preemptive snapshot d55922f1 at index 27
I170801 22:39:22.720272 63503 storage/store.go:3486  [replicate,n1,s1,r7/1:/Table/1{2-3}] streamed snapshot to (n3,s3):?: kv pairs: 39, log entries: 17, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.720555 65343 storage/replica_raftstorage.go:718  [n3,s3,r7/?:{-}] applying preemptive snapshot at index 27 (id=d55922f1, encoded size=21849, 1 rocksdb batches, 17 log entries)
I170801 22:39:22.721291 65343 storage/replica_raftstorage.go:726  [n3,s3,r7/?:/Table/1{2-3}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.721699 63503 storage/replica_command.go:3600  [replicate,n1,s1,r7/1:/Table/1{2-3}] change replicas (ADD_REPLICA (n3,s3):2): read existing descriptor r7:/Table/1{2-3} [(n1,s1):1, next=2]
I170801 22:39:22.725610 65415 storage/replica.go:2937  [n1,s1,r7/1:/Table/1{2-3}] proposing ADD_REPLICA((n3,s3):2): updated=[(n1,s1):1 (n3,s3):2] next=3
I170801 22:39:22.726060 63503 storage/queue.go:723  [n1,replicate] purgatory is now empty
I170801 22:39:22.726225 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r10/1:/Table/1{5-6}] generated preemptive snapshot 02473219 at index 22
I170801 22:39:22.726870 62879 storage/store.go:3486  [replicate,n1,s1,r10/1:/Table/1{5-6}] streamed snapshot to (n3,s3):?: kv pairs: 10, log entries: 12, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.727167 65185 storage/replica_raftstorage.go:718  [n3,s3,r10/?:{-}] applying preemptive snapshot at index 22 (id=02473219, encoded size=7757, 1 rocksdb batches, 12 log entries)
I170801 22:39:22.727767 65185 storage/replica_raftstorage.go:726  [n3,s3,r10/?:/Table/1{5-6}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.728250 62879 storage/replica_command.go:3600  [replicate,n1,s1,r10/1:/Table/1{5-6}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r10:/Table/1{5-6} [(n1,s1):1, (n4,s4):2, next=3]
I170801 22:39:22.735057 65443 storage/replica.go:2937  [n1,s1,r10/1:/Table/1{5-6}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n4,s4):2 (n3,s3):3] next=4
I170801 22:39:22.736055 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r9/1:/Table/1{4-5}] generated preemptive snapshot bc17af33 at index 22
I170801 22:39:22.736724 62879 storage/store.go:3486  [replicate,n1,s1,r9/1:/Table/1{4-5}] streamed snapshot to (n5,s5):?: kv pairs: 10, log entries: 12, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.737037 65466 storage/replica_raftstorage.go:718  [n5,s5,r9/?:{-}] applying preemptive snapshot at index 22 (id=bc17af33, encoded size=7754, 1 rocksdb batches, 12 log entries)
I170801 22:39:22.737534 65466 storage/replica_raftstorage.go:726  [n5,s5,r9/?:/Table/1{4-5}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.737840 62879 storage/replica_command.go:3600  [replicate,n1,s1,r9/1:/Table/1{4-5}] change replicas (ADD_REPLICA (n5,s5):3): read existing descriptor r9:/Table/1{4-5} [(n1,s1):1, (n4,s4):2, next=3]
I170801 22:39:22.741919 65420 storage/replica.go:2937  [n1,s1,r9/1:/Table/1{4-5}] proposing ADD_REPLICA((n5,s5):3): updated=[(n1,s1):1 (n4,s4):2 (n5,s5):3] next=4
I170801 22:39:22.743151 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] generated preemptive snapshot b01592e7 at index 29
I170801 22:39:22.743936 62879 storage/store.go:3486  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] streamed snapshot to (n4,s4):?: kv pairs: 51, log entries: 19, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.744305 65422 storage/replica_raftstorage.go:718  [n4,s4,r5/?:{-}] applying preemptive snapshot at index 29 (id=b01592e7, encoded size=28405, 1 rocksdb batches, 19 log entries)
I170801 22:39:22.744945 65422 storage/replica_raftstorage.go:726  [n4,s4,r5/?:/Table/{SystemCon���-11}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.745461 62879 storage/replica_command.go:3600  [replicate,n1,s1,r5/1:/Table/{SystemCon���-11}] change replicas (ADD_REPLICA (n4,s4):3): read existing descriptor r5:/Table/{SystemConfigSpan/Start-11} [(n1,s1):1, (n5,s5):2, next=3]
I170801 22:39:22.750162 65506 storage/replica.go:2937  [n1,s1,r5/1:/Table/{SystemCon���-11}] proposing ADD_REPLICA((n4,s4):3): updated=[(n1,s1):1 (n5,s5):2 (n4,s4):3] next=4
I170801 22:39:22.751668 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r14/1:/{Table/19-Max}] generated preemptive snapshot bc46fbe0 at index 14
I170801 22:39:22.752249 62879 storage/store.go:3486  [replicate,n1,s1,r14/1:/{Table/19-Max}] streamed snapshot to (n2,s2):?: kv pairs: 9, log entries: 4, rate-limit: 8.0 MiB/sec, 0ms
I170801 22:39:22.752527 65447 storage/replica_raftstorage.go:718  [n2,s2,r14/?:{-}] applying preemptive snapshot at index 14 (id=bc46fbe0, encoded size=2507, 1 rocksdb batches, 4 log entries)
I170801 22:39:22.752869 65447 storage/replica_raftstorage.go:726  [n2,s2,r14/?:/{Table/19-Max}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.753248 62879 storage/replica_command.go:3600  [replicate,n1,s1,r14/1:/{Table/19-Max}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r14:/{Table/19-Max} [(n1,s1):1, (n3,s3):2, next=3]
I170801 22:39:22.757442 65531 storage/replica.go:2937  [n1,s1,r14/1:/{Table/19-Max}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170801 22:39:22.758738 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r12/1:/Table/1{7-8}] generated preemptive snapshot 4568bd59 at index 19
I170801 22:39:22.759250 62879 storage/store.go:3486  [replicate,n1,s1,r12/1:/Table/1{7-8}] streamed snapshot to (n4,s4):?: kv pairs: 10, log entries: 9, rate-limit: 8.0 MiB/sec, 0ms
I170801 22:39:22.759483 65374 storage/replica_raftstorage.go:718  [n4,s4,r12/?:{-}] applying preemptive snapshot at index 19 (id=4568bd59, encoded size=6076, 1 rocksdb batches, 9 log entries)
I170801 22:39:22.760049 65374 storage/replica_raftstorage.go:726  [n4,s4,r12/?:/Table/1{7-8}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.761112 62879 storage/replica_command.go:3600  [replicate,n1,s1,r12/1:/Table/1{7-8}] change replicas (ADD_REPLICA (n4,s4):3): read existing descriptor r12:/Table/1{7-8} [(n1,s1):1, (n2,s2):2, next=3]
I170801 22:39:22.765589 65469 storage/replica.go:2937  [n1,s1,r12/1:/Table/1{7-8}] proposing ADD_REPLICA((n4,s4):3): updated=[(n1,s1):1 (n2,s2):2 (n4,s4):3] next=4
I170801 22:39:22.766866 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r6/1:/Table/1{1-2}] generated preemptive snapshot 38c21dbe at index 19
I170801 22:39:22.767452 62879 storage/store.go:3486  [replicate,n1,s1,r6/1:/Table/1{1-2}] streamed snapshot to (n5,s5):?: kv pairs: 10, log entries: 9, rate-limit: 8.0 MiB/sec, 0ms
I170801 22:39:22.768049 65495 storage/replica_raftstorage.go:718  [n5,s5,r6/?:{-}] applying preemptive snapshot at index 19 (id=38c21dbe, encoded size=7330, 1 rocksdb batches, 9 log entries)
I170801 22:39:22.768571 65495 storage/replica_raftstorage.go:726  [n5,s5,r6/?:/Table/1{1-2}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.768907 62879 storage/replica_command.go:3600  [replicate,n1,s1,r6/1:/Table/1{1-2}] change replicas (ADD_REPLICA (n5,s5):3): read existing descriptor r6:/Table/1{1-2} [(n1,s1):1, (n2,s2):2, next=3]
I170801 22:39:22.773243 65576 storage/replica.go:2937  [n1,s1,r6/1:/Table/1{1-2}] proposing ADD_REPLICA((n5,s5):3): updated=[(n1,s1):1 (n2,s2):2 (n5,s5):3] next=4
I170801 22:39:22.774466 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot 51317b73 at index 108
I170801 22:39:22.775305 62879 storage/store.go:3486  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n3,s3):?: kv pairs: 63, log entries: 45, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.775577 65586 storage/replica_raftstorage.go:718  [n3,s3,r1/?:{-}] applying preemptive snapshot at index 108 (id=51317b73, encoded size=25347, 1 rocksdb batches, 45 log entries)
I170801 22:39:22.776813 65586 storage/replica_raftstorage.go:726  [n3,s3,r1/?:/{Min-System/}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170801 22:39:22.777226 62879 storage/replica_command.go:3600  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r1:/{Min-System/} [(n1,s1):1, (n5,s5):2, next=3]
I170801 22:39:22.781389 65581 storage/replica.go:2937  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n5,s5):2 (n3,s3):3] next=4
I170801 22:39:22.783037 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r2/1:/System/{-tsd}] generated preemptive snapshot 2d84aaa0 at index 55
I170801 22:39:22.784320 62879 storage/store.go:3486  [replicate,n1,s1,r2/1:/System/{-tsd}] streamed snapshot to (n2,s2):?: kv pairs: 44, log entries: 5, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.784633 65606 storage/replica_raftstorage.go:718  [n2,s2,r2/?:{-}] applying preemptive snapshot at index 55 (id=2d84aaa0, encoded size=98455, 1 rocksdb batches, 5 log entries)
I170801 22:39:22.785620 65606 storage/replica_raftstorage.go:726  [n2,s2,r2/?:/System/{-tsd}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170801 22:39:22.785983 62879 storage/replica_command.go:3600  [replicate,n1,s1,r2/1:/System/{-tsd}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r2:/System/{-tsd} [(n1,s1):1, (n4,s4):2, next=3]
I170801 22:39:22.790454 65651 storage/replica.go:2937  [n1,s1,r2/1:/System/{-tsd}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n4,s4):2 (n2,s2):3] next=4
I170801 22:39:22.791739 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r3/1:/System/ts{d-e}] generated preemptive snapshot fa67704e at index 26
I170801 22:39:22.799243 62879 storage/store.go:3486  [replicate,n1,s1,r3/1:/System/ts{d-e}] streamed snapshot to (n4,s4):?: kv pairs: 1422, log entries: 6, rate-limit: 8.0 MiB/sec, 7ms
I170801 22:39:22.799561 65514 storage/replica_raftstorage.go:718  [n4,s4,r3/?:{-}] applying preemptive snapshot at index 26 (id=fa67704e, encoded size=204993, 1 rocksdb batches, 6 log entries)
I170801 22:39:22.801064 65514 storage/replica_raftstorage.go:726  [n4,s4,r3/?:/System/ts{d-e}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170801 22:39:22.801455 62879 storage/replica_command.go:3600  [replicate,n1,s1,r3/1:/System/ts{d-e}] change replicas (ADD_REPLICA (n4,s4):3): read existing descriptor r3:/System/ts{d-e} [(n1,s1):1, (n2,s2):2, next=3]
I170801 22:39:22.805634 65700 storage/replica.go:2937  [n1,s1,r3/1:/System/ts{d-e}] proposing ADD_REPLICA((n4,s4):3): updated=[(n1,s1):1 (n2,s2):2 (n4,s4):3] next=4
I170801 22:39:22.806918 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] generated preemptive snapshot fd52ea70 at index 26
I170801 22:39:22.807507 62879 storage/store.go:3486  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] streamed snapshot to (n2,s2):?: kv pairs: 12, log entries: 16, rate-limit: 8.0 MiB/sec, 0ms
I170801 22:39:22.807708 65613 storage/replica_raftstorage.go:718  [n2,s2,r4/?:{-}] applying preemptive snapshot at index 26 (id=fd52ea70, encoded size=12179, 1 rocksdb batches, 16 log entries)
I170801 22:39:22.808230 65613 storage/replica_raftstorage.go:726  [n2,s2,r4/?:/{System/tse-Table/System���}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.808603 62879 storage/replica_command.go:3600  [replicate,n1,s1,r4/1:/{System/tse-Table/System���}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r4:/{System/tse-Table/SystemConfigSpan/Start} [(n1,s1):1, (n5,s5):2, next=3]
I170801 22:39:22.813334 65595 storage/replica.go:2937  [n1,s1,r4/1:/{System/tse-Table/System���}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n5,s5):2 (n2,s2):3] next=4
I170801 22:39:22.814794 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r7/1:/Table/1{2-3}] generated preemptive snapshot e9c7287a at index 30
I170801 22:39:22.815414 62879 storage/store.go:3486  [replicate,n1,s1,r7/1:/Table/1{2-3}] streamed snapshot to (n4,s4):?: kv pairs: 40, log entries: 20, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.815677 65690 storage/replica_raftstorage.go:718  [n4,s4,r7/?:{-}] applying preemptive snapshot at index 30 (id=e9c7287a, encoded size=23880, 1 rocksdb batches, 20 log entries)
I170801 22:39:22.816304 65690 storage/replica_raftstorage.go:726  [n4,s4,r7/?:/Table/1{2-3}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.816656 62879 storage/replica_command.go:3600  [replicate,n1,s1,r7/1:/Table/1{2-3}] change replicas (ADD_REPLICA (n4,s4):3): read existing descriptor r7:/Table/1{2-3} [(n1,s1):1, (n3,s3):2, next=3]
I170801 22:39:22.820715 65691 storage/replica.go:2937  [n1,s1,r7/1:/Table/1{2-3}] proposing ADD_REPLICA((n4,s4):3): updated=[(n1,s1):1 (n3,s3):2 (n4,s4):3] next=4
I170801 22:39:22.822675 64082 storage/replica_proposal.go:453  [n2,s2,r4/3:/{System/tse-Table/System���}] new range lease repl=(n2,s2):3 start=1501627162.821751717,0 epo=1 pro=1501627162.821754417,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1501627171.390678381,0 pro=1501627162.390694081,0
I170801 22:39:22.822850 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r14/1:/{Table/19-Max}] generated preemptive snapshot 9c5dad11 at index 17
I170801 22:39:22.823628 62879 storage/store.go:3486  [replicate,n1,s1,r14/1:/{Table/19-Max}] streamed snapshot to (n5,s5):?: kv pairs: 10, log entries: 7, rate-limit: 2.0 MiB/sec, 1ms
I170801 22:39:22.823839 65737 storage/replica_raftstorage.go:718  [n5,s5,r14/?:{-}] applying preemptive snapshot at index 17 (id=9c5dad11, encoded size=4574, 1 rocksdb batches, 7 log entries)
I170801 22:39:22.824315 65737 storage/replica_raftstorage.go:726  [n5,s5,r14/?:/{Table/19-Max}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.824776 62879 storage/replica_command.go:3600  [replicate,n1,s1,r14/1:/{Table/19-Max}] change replicas (ADD_REPLICA (n5,s5):4): read existing descriptor r14:/{Table/19-Max} [(n1,s1):1, (n3,s3):2, (n2,s2):3, next=4]
I170801 22:39:22.829344 65672 storage/replica.go:2937  [n1,s1,r14/1:/{Table/19-Max}] proposing ADD_REPLICA((n5,s5):4): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3 (n5,s5):4] next=5
I170801 22:39:22.830492 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r10/1:/Table/1{5-6}] generated preemptive snapshot 730808ff at index 25
I170801 22:39:22.831399 65767 storage/replica_raftstorage.go:718  [n5,s5,r10/?:{-}] applying preemptive snapshot at index 25 (id=730808ff, encoded size=9820, 1 rocksdb batches, 15 log entries)
I170801 22:39:22.831556 62879 storage/store.go:3486  [replicate,n1,s1,r10/1:/Table/1{5-6}] streamed snapshot to (n5,s5):?: kv pairs: 11, log entries: 15, rate-limit: 2.0 MiB/sec, 1ms
I170801 22:39:22.831899 65767 storage/replica_raftstorage.go:726  [n5,s5,r10/?:/Table/1{5-6}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.832266 62879 storage/replica_command.go:3600  [replicate,n1,s1,r10/1:/Table/1{5-6}] change replicas (ADD_REPLICA (n5,s5):4): read existing descriptor r10:/Table/1{5-6} [(n1,s1):1, (n4,s4):2, (n3,s3):3, next=4]
I170801 22:39:22.836823 65782 storage/replica.go:2937  [n1,s1,r10/1:/Table/1{5-6}] proposing ADD_REPLICA((n5,s5):4): updated=[(n1,s1):1 (n4,s4):2 (n3,s3):3 (n5,s5):4] next=5
I170801 22:39:22.838287 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r13/1:/Table/1{8-9}] generated preemptive snapshot 8f9ab412 at index 19
I170801 22:39:22.839098 62879 storage/store.go:3486  [replicate,n1,s1,r13/1:/Table/1{8-9}] streamed snapshot to (n4,s4):?: kv pairs: 10, log entries: 9, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.839375 65661 storage/replica_raftstorage.go:718  [n4,s4,r13/?:{-}] applying preemptive snapshot at index 19 (id=8f9ab412, encoded size=6076, 1 rocksdb batches, 9 log entries)
I170801 22:39:22.839729 65661 storage/replica_raftstorage.go:726  [n4,s4,r13/?:/Table/1{8-9}] applied preemptive snapshot in 0ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.840137 62879 storage/replica_command.go:3600  [replicate,n1,s1,r13/1:/Table/1{8-9}] change replicas (ADD_REPLICA (n4,s4):3): read existing descriptor r13:/Table/1{8-9} [(n1,s1):1, (n3,s3):2, next=3]
I170801 22:39:22.844925 65783 storage/replica.go:2937  [n1,s1,r13/1:/Table/1{8-9}] proposing ADD_REPLICA((n4,s4):3): updated=[(n1,s1):1 (n3,s3):2 (n4,s4):3] next=4
I170801 22:39:22.846096 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r8/1:/Table/1{3-4}] generated preemptive snapshot 6d56ac0d at index 93
I170801 22:39:22.847176 62879 storage/store.go:3486  [replicate,n1,s1,r8/1:/Table/1{3-4}] streamed snapshot to (n2,s2):?: kv pairs: 238, log entries: 22, rate-limit: 8.0 MiB/sec, 1ms
I170801 22:39:22.847484 65785 storage/replica_raftstorage.go:718  [n2,s2,r8/?:{-}] applying preemptive snapshot at index 93 (id=6d56ac0d, encoded size=47554, 1 rocksdb batches, 22 log entries)
I170801 22:39:22.848385 65785 storage/replica_raftstorage.go:726  [n2,s2,r8/?:/Table/1{3-4}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.848727 62879 storage/replica_command.go:3600  [replicate,n1,s1,r8/1:/Table/1{3-4}] change replicas (ADD_REPLICA (n2,s2):3): read existing descriptor r8:/Table/1{3-4} [(n1,s1):1, (n3,s3):2, next=3]
I170801 22:39:22.852878 65812 storage/replica.go:2937  [n1,s1,r8/1:/Table/1{3-4}] proposing ADD_REPLICA((n2,s2):3): updated=[(n1,s1):1 (n3,s3):2 (n2,s2):3] next=4
I170801 22:39:22.854110 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r1/1:/{Min-System/}] generated preemptive snapshot 6dbaea15 at index 127
I170801 22:39:22.855344 62879 storage/store.go:3486  [replicate,n1,s1,r1/1:/{Min-System/}] streamed snapshot to (n2,s2):?: kv pairs: 74, log entries: 64, rate-limit: 2.0 MiB/sec, 1ms
I170801 22:39:22.855807 65744 storage/replica_raftstorage.go:718  [n2,s2,r1/?:{-}] applying preemptive snapshot at index 127 (id=6dbaea15, encoded size=35779, 1 rocksdb batches, 64 log entries)
I170801 22:39:22.857442 65744 storage/replica_raftstorage.go:726  [n2,s2,r1/?:/{Min-System/}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170801 22:39:22.857741 62879 storage/replica_command.go:3600  [replicate,n1,s1,r1/1:/{Min-System/}] change replicas (ADD_REPLICA (n2,s2):4): read existing descriptor r1:/{Min-System/} [(n1,s1):1, (n5,s5):2, (n3,s3):3, next=4]
I170801 22:39:22.861473 65868 storage/replica.go:2937  [n1,s1,r1/1:/{Min-System/}] proposing ADD_REPLICA((n2,s2):4): updated=[(n1,s1):1 (n5,s5):2 (n3,s3):3 (n2,s2):4] next=5
I170801 22:39:22.862985 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r11/1:/Table/1{6-7}] generated preemptive snapshot 612af941 at index 22
I170801 22:39:22.863617 62879 storage/store.go:3486  [replicate,n1,s1,r11/1:/Table/1{6-7}] streamed snapshot to (n3,s3):?: kv pairs: 11, log entries: 12, rate-limit: 2.0 MiB/sec, 0ms
I170801 22:39:22.863867 65804 storage/replica_raftstorage.go:718  [n3,s3,r11/?:{-}] applying preemptive snapshot at index 22 (id=612af941, encoded size=8140, 1 rocksdb batches, 12 log entries)
I170801 22:39:22.864418 65804 storage/replica_raftstorage.go:726  [n3,s3,r11/?:/Table/1{6-7}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.864755 62879 storage/replica_command.go:3600  [replicate,n1,s1,r11/1:/Table/1{6-7}] change replicas (ADD_REPLICA (n3,s3):4): read existing descriptor r11:/Table/1{6-7} [(n1,s1):1, (n2,s2):2, (n5,s5):3, next=4]
I170801 22:39:22.874375 64110 storage/replica_raftstorage.go:509  [replicate,n2,s2,r4/3:/{System/tse-Table/System���}] generated preemptive snapshot 13ea5996 at index 30
I170801 22:39:22.875528 65871 storage/replica.go:2937  [n1,s1,r11/1:/Table/1{6-7}] proposing ADD_REPLICA((n3,s3):4): updated=[(n1,s1):1 (n2,s2):2 (n5,s5):3 (n3,s3):4] next=5
I170801 22:39:22.879478 64060 storage/replica_proposal.go:453  [n2,s2,r11/2:/Table/1{6-7}] new range lease repl=(n2,s2):2 start=1501627162.878210135,0 epo=1 pro=1501627162.878212035,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1501627171.390678381,0 pro=1501627162.390694081,0
I170801 22:39:22.881079 64899 storage/replica_proposal.go:453  [n5,s5,r14/4:/{Table/19-Max}] new range lease repl=(n5,s5):4 start=1501627162.879650716,0 epo=1 pro=1501627162.879652515,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1501627171.390678381,0 pro=1501627162.390694081,0
I170801 22:39:22.881898 64309 storage/replica_proposal.go:453  [n3,s3,r10/3:/Table/1{5-6}] new range lease repl=(n3,s3):3 start=1501627162.880699001,0 epo=1 pro=1501627162.880700801,0 following repl=(n1,s1):1 start=0.000000000,0 exp=1501627171.390678381,0 pro=1501627162.390694081,0
I170801 22:39:22.882175 65914 storage/raft_transport.go:451  [n5] raft transport stream to node 3 established
E170801 22:39:22.882700 64354 storage/queue.go:656  [replicate,n3,s3,r10/3:/Table/1{5-6}] must supply at least one replica to allocator.RemoveTarget()
I170801 22:39:22.886318 62879 storage/replica_raftstorage.go:509  [replicate,n1,s1,r12/1:/Table/1{7-8}] generated preemptive snapshot eb2fdbcc at index 22
I170801 22:39:22.887779 62879 storage/store.go:3486  [replicate,n1,s1,r12/1:/Table/1{7-8}] streamed snapshot to (n3,s3):?: kv pairs: 11, log entries: 12, rate-limit: 2.0 MiB/sec, 1ms
I170801 22:39:22.888144 65945 storage/replica_raftstorage.go:718  [n3,s3,r12/?:{-}] applying preemptive snapshot at index 22 (id=eb2fdbcc, encoded size=8139, 1 rocksdb batches, 12 log entries)
I170801 22:39:22.888712 65945 storage/replica_raftstorage.go:726  [n3,s3,r12/?:/Table/1{7-8}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170801 22:39:22.889097 62879 storage/replica_command.go:
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/client_replica_test.go
+++ b/pkg/storage/client_replica_test.go
@@ -1512,8 +1512,6 @@ func TestDrainRangeRejection(t *testing.T) {
 func TestSystemZoneConfigs(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
-	t.Skip("https://github.com/cockroachdb/cockroach/issues/17375")
-
 	if testing.Short() {
 		t.Skip("short flag")
 	}

--- a/pkg/storage/replica.go
+++ b/pkg/storage/replica.go
@@ -3343,6 +3343,9 @@ func (r *Replica) handleRaftReadyRaftMuLocked(
 	r.mu.lastTerm = lastTerm
 	r.mu.raftLogSize = raftLogSize
 	r.mu.leaderID = leaderID
+	if r.mu.replicaID == leaderID && !raft.IsEmptyHardState(rd.HardState) {
+		r.setEstimatedCommitIndexLocked(rd.HardState.Commit)
+	}
 	r.mu.Unlock()
 
 	for _, message := range rd.Messages {

